crates/medmodels-core/src/medrecord/example_dataset/mod.rs

```diff
@@ -286,4 +601,39 @@ mod test {
                 .count()
         );
     }
+
+    #[test]
+    fn test_from_advanced_example_dataset() {
+        let medrecord = MedRecord::from_advanced_example_dataset();
+
+        assert_eq!(1415, medrecord.node_count());
+        assert_eq!(26854, medrecord.edge_count());
+
+        assert_eq!(
+            608,
+            medrecord
+                .nodes_in_group(&"diagnosis".into())
+                .unwrap()
+                .count()
+        );
+        assert_eq!(
+            43,
+            medrecord.nodes_in_group(&"drug".into()).unwrap().count()
+        );
+        assert_eq!(
+            600,
+            medrecord.nodes_in_group(&"patient".into()).unwrap().count()
+        );
+        assert_eq!(
+            1,
+            medrecord.nodes_in_group(&"event".into()).unwrap().count()
+        );
+        assert_eq!(
+            163,
+            medrecord
+                .nodes_in_group(&"procedure".into())
+                .unwrap()
+                .count()
+        );
+    }
```

Comment:
could you also add tests for the count of edges in groups?
E.g.
```rust
assert_eq!(
    some_number,
    medrecord
        .edges_in_group(&"patient_diagnosis".into())
        .unwrap()
        .count()
);
```

this would also need to be implemented for `test_from_simple_example_dataset`